recipes/upstream/google-cloud/gke/variables.tf

```diff
@@ -56,50 +52,39 @@ variable "region" {
   }
 }
 
-variable "ip_cidr_range" {
-  type        = string
-  default     = "10.10.0.0/24"
-  description = "Range of private IPs available for the Google Subnet"
-}
+# variable "ip_cidr_range" {}
```

Comment:
We could copy these as-is from the module so they can be used

Reply:
As I wrote all the repos, I left the minimum values ​​necessary for deployment uncommented, while still leaving all the possible usable values ​​commented out.
If you have a directive for this, I will implement it.
As an example:
- leave the minimal variables, but delete the unused ones;
- leave all the variables available with the default values ​​configured both in the main and in the module;
- etc.

Let me know.

Reply:
I think taking option 2 would be best within reason, keeping all options available for direct use, without having to update the variables.tf and uncomment or populate

One main benefit is this would make the docs.md of recipes more of a useful reference

This can be a separate issue/PR to review all recipes if you feel it's better to keep separate, but I _think_ other recipes are already doing this, or pretty close

Reply:
The docs to refer to should be the modules; there are all the available fields with the available resources.
The recipe main contains only what needs to be modified; that's why everything is commented on.
Anyway, you tell me. This is a style choice and I adapt to what you tell me.

Reply:
Agree with Derek. I'd rather the end user only update the terraform.tfvars and not have to uncomment code in the main.tf and variables.tf. That also makes it more difficult for users to pull in new changes.